diff -rupN calf-0.90.7.old/CMakeLists.txt calf-0.90.7/CMakeLists.txt
--- calf-0.90.7.old/CMakeLists.txt	2025-05-10 15:30:02.000000000 +0200
+++ calf-0.90.7/CMakeLists.txt	2025-07-08 16:25:36.117191739 +0200
@@ -125,7 +125,7 @@ endif()
 
 if(WANT_LASH)
     if(PKG_CONFIG_FOUND)
-        pkg_search_module(LASH lash-1.0)
+        pkg_search_module(LASH liblash)
     endif()
     if(LASH_FOUND)
         include_directories(${LASH_INCLUDE_DIRS})
diff -rupN calf-0.90.7.old/src/session_mgr.cpp calf-0.90.7/src/session_mgr.cpp
--- calf-0.90.7.old/src/session_mgr.cpp	2025-05-10 15:30:02.000000000 +0200
+++ calf-0.90.7/src/session_mgr.cpp	2025-07-08 16:26:40.714398005 +0200
@@ -25,7 +25,7 @@
 
 #include <calf/utils.h>
 #include <gtk/gtk.h>
-#include <lash/lash.h>
+#include <lash-1.0/lash/lash.h>
 #include <glib.h>
 #include <string.h>
 
